examples/complete/variables.tf

```diff
@@ -45,3 +39,9 @@ variable "existing_sm_instance_region" {
   description = "The region of the existing Secrets Manager instance. Only required if value is passed into var.existing_sm_instance_guid"
   default     = null
 }
+
+variable "service_endpoints" {
```

Comment:
no need to expose this as a variable, just add it directly in main.tf of the example